packages/valory/skills/check_stop_trading_abci/rounds.py

```diff
@@ -135,20 +130,20 @@ class CheckStopTradingAbciApp(AbciApp[Event]):  # pylint: disable=too-few-public
             Event.NONE: CheckStopTradingRound,
             Event.ROUND_TIMEOUT: CheckStopTradingRound,
             Event.NO_MAJORITY: CheckStopTradingRound,
-            Event.SKIP_TRADING: FinishedCheckStopTradingWithSkipTradingRound,
+            Event.SKIP_TRADING: FinishedWithSkipTradingRound,
         },
         FinishedCheckStopTradingRound: {},
-        FinishedCheckStopTradingWithSkipTradingRound: {},
+        FinishedWithSkipTradingRound: {},
     }
     final_states: Set[AppState] = {
         FinishedCheckStopTradingRound,
-        FinishedCheckStopTradingWithSkipTradingRound,
+        FinishedWithSkipTradingRound,
     }
     event_to_timeout: Dict[Event, float] = {
         Event.ROUND_TIMEOUT: 30.0,
     }
     db_pre_conditions: Dict[AppState, Set[str]] = {CheckStopTradingRound: set()}
     db_post_conditions: Dict[AppState, Set[str]] = {
         FinishedCheckStopTradingRound: set(),
-        FinishedCheckStopTradingWithSkipTradingRound: set(),
+        FinishedWithSkipTradingRound: set(),
```

Comment:
How about `ResumeTradingRound` and `StopTradingRound`? Easier to read the composition.

Reply:
Because this is computed on each iteration: on each iteration it's decided whether to go to the "trading branch" and whether skip the trading branch. "Resume" is not an approppriate name here, IMO (it can simply always trade, so it loses the meaning of "resume").